client/src/lib/utils/insertMessage.ts

```diff
@@ -0,0 +1,63 @@
+
+import { createPublicThread }  from "./createPublicThread";
+import { MessageCardProps } from "../../ui/messaging/MessageCard";
+import { Profile } from "../store/profile";
+import { createSupabaseClient } from "../supabase/client"
+
+
+export async function insertMessage(newMessage: MessageCardProps, conversationId: string, profile: Profile) 
+{
+    const supabase = createSupabaseClient()  
+
+    if (conversationId && newMessage) {
```

Comment:
nit: Why does these values need to be checked if it's not null? From the parameters it's expecting it to non-null values